searchdepend.mlg

```diff
@@ -42,15 +42,15 @@ let add_inductive ((k,i):Names.inductive)(d:Data.t) =
 let add_constructor(((k,i),j):Names.constructor)(d:Data.t) =
   Data.add (Names.GlobRef.ConstructRef ((k,i),j)) d
 
-let collect_long_names (c:Constr.t) (acc:Data.t) =
+let collect_long_names avoid (c:Constr.t) (acc:Data.t) =
```

Comment:
Why is this a list instead of an option?

Reply:
I started from the idea that we could have nested inductive types to avoid. At the end, there is no nesting and there is at most one.